tests/unit/serialization/custom_global_serialization_test.py

```diff
@@ -128,3 +139,21 @@ def test_double_register_custom_serializer(self):
 
         with self.assertRaises(ValueError):
             service._registry.safe_register_serializer(TheOtherCustomSerializer, CustomClass)
+
+
+    def test_class(self):
+
+        config = _Config()
+
+        config.custom_serializers = {CustomClass: CustomSerializer, TheOtherCustomClass: CustomSerializer}
```

Comment:
Only one of them would suffice here

Reply:
Why? They have same superclass but they are different. That is what I thought.